openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -251,6 +251,7 @@ def cleanup_placeholder(self, placeholder, failed):
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        reset_frame_range(playback=False, render=False, fps=False)
```

Comment:
Should we add the `instances=True` argument here just to make clear by reading only this line that the sole focus is on resetting the instances. Just from reading this line (without knowing the function definition) it's not clear that the intent is to only update the instances in the scene.

Reply:
The option says "_If enabled, the frame range and the handles of all the publishable instances will be updated when using the 'Reset Frame Range' functionality_" so it does not concern the creation of the workfile.

Reply:
> so it does not concern the creation of the workfile.

Well, actually it does. Because that settings toggle influences this particular call for the workfile template too. If I disable that option in the settings than the workfile template wouldn't get the instances reset either. Not saying it's a problem in the end, just something to be wary of. I'll resolve the conversation leaving it at that.

Reply:
You're absolutely right, I hadn't seen it that way, thank you.

Reply:
I changed the logic so that the setting "_update instances on reset frame range_" has no influence on the templates.